production-swarm/docker-stack.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+services:
+  jobservice:
+    env_file:
+      - ./rabbitmq.env
+      - ./postgres.env
+      - ./jobservice.env
+    environment:
+      CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
```

Comment:
I'd be maybe tempted to at least match what the marathon template is doing for consistency, especially given you are doing the same thing for other settings.